### PR TITLE
Merge appcast into Build CI

### PR DESCRIPTION
**Pull request by SkyrilHD**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -71,28 +71,24 @@ jobs:
...
       - name: Get Sparkle 1.26
+        if: github.event_name == 'release' && github.repository_owner == 'acidanthera'
         run: |
           curl -L -s "https://github.com/sparkle-project/Sparkle/releases/download/1.26.0/Sparkle-1.26.0.tar.xz" -o Sparkle.tar.xz || exit 1
           tar -xf Sparkle.tar.xz || exit 1
 
       - name: Get Information & Sign
+        if: github.event_name == 'release' && github.repository_owner == 'acidanthera'
+        env:
+          SIGNATURE_FOR_SIGNING: ${{ secrets.SIGNATURE_FOR_SIGNING }}
```
Comment (vit9696) [inline .github/workflows/main.yml]: Hm, this one looks unused.

**Comment by vit9696** [comment]
Could you change:
- `github.repository_owner == 'acidanthera'` to `github.repository_owner == 'acidanthera' && github.event_name != 'pull_request'`
- `github.repository_owner != 'acidanthera'` to `github.repository_owner != 'acidanthera' || github.event_name == 'pull_request'`

In other steps? Otherwise the CI will yell at this PR.

File: .github/workflows/main.yml (hunk 1 above)
Comment (SkyrilHD) [reply .github/workflows/main.yml]: yep. removed

**Comment by SkyrilHD** [comment]
> Could you change:
> 
> * `github.repository_owner == 'acidanthera'` to `github.repository_owner == 'acidanthera' && github.event_name != 'pull_request'`
> * `github.repository_owner != 'acidanthera'` to `github.repository_owner != 'acidanthera' || github.event_name == 'pull_request'`
> 
> In other steps? Otherwise the CI will yell at this PR.

Added.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -32,67 +32,61 @@ jobs:
...
-        if: github.repository_owner == 'acidanthera'
+        if: github.repository_owner == 'acidanthera' && github.event_name != 'pull_request'
         uses: actions/upload-artifact@v2
         with:
           name: Artifacts
           path: build/*/*.dmg
 
       - name: Upload to Release
-        if: github.event_name == 'release' && github.repository_owner == 'acidanthera'
+        if: github.event_name == 'release' && github.repository_owner == 'acidanthera' && github.event_name != 'pull_request'
```
Comment (vit9696) [inline .github/workflows/main.yml]: This one is always true, no need to check for `pull_request` here and below where there is a `release` check already.

**Comment by vit9696** [comment]
Seems to be working well, thanks for your effort!

**Pull request merged**